From: oneukum@suse.de
Patch-mainline: v2.6.38-rc8
References: bnc#670864
Date: Mi 16. Feb 09:46:36 CET 2011
Subject: blacklist another BT device that cannot operate without firmware
Git-commit: e9036e336a8e5640871e0006ea4a89982b25046f

Add BT device blacklist 0489:e02c (Atheros AR5BBU12)
for ath3k firmware supported.

Acked-by: Lee, Chun-Yi <jlee@novell.com>
Signed-off-by: Cho, Yu-Chen <acho@novell.com>

---
 drivers/bluetooth/btusb.c |    3 +++
 1 file changed, 3 insertions(+)

--- a/drivers/bluetooth/btusb.c
+++ b/drivers/bluetooth/btusb.c
@@ -96,6 +96,9 @@ static struct usb_device_id blacklist_ta
 
         /* Atheros Malbec with sflash firmware */
         { USB_DEVICE(0x03f0, 0x311d), .driver_info = BTUSB_IGNORE },
+	
+	/* Atheros AR5BBU12 with sflash firmware */
+	{ USB_DEVICE(0x0489, 0xe02c), .driver_info = BTUSB_IGNORE },
 
 	/* Broadcom BCM2035 */
 	{ USB_DEVICE(0x0a5c, 0x2035), .driver_info = BTUSB_WRONG_SCO_MTU },


